accounting/bank/models.py

```diff
@@ -16,32 +16,26 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-
+# pylint: disable=unsubscriptable-object
```

Comment:
@UmanShahzad What part of the code is this related to? Could you put it next to the false positive, instead of globally for the file, since that might end up hiding real issues elsewhere in the file, now or in the future?

Reply:
Yep, it was introduced after using the PostgreSQL JSONField. I'll make it explicit.